pythonforandroid/toolchain.py

```diff
@@ -78,7 +78,7 @@ def __getattr__(self, key):
 sys.path.insert(0, join(toolchain_dir, "tools", "external"))
 
 
-DEFAULT_ANDROID_API = 15
+DEFAULT_ANDROID_API = 16
```

Comment:
Sorry this has nothing to do with the typo erros. It's just an adaption to my development situation.